client/actions/planning/notifications.js

```diff
@@ -44,30 +44,35 @@ const onPlanningCreated = (_e, data) => (
 const onPlanningUpdated = (_e, data) => (
     (dispatch, getState) => {
         if (get(data, 'item')) {
-            dispatch(planning.ui.scheduleRefetch())
-                .then((results) => {
-                    const selectedItems = selectors.multiSelect.selectedPlannings(getState());
-                    const currentPreviewId = selectors.main.previewId(getState());
-                    const currentEditId = selectors.forms.currentItemId(getState());
-
-                    const loadedFromRefetch = selectedItems.indexOf(data.item) !== -1 &&
-                        !get(results, '[0]._items').find((plan) => plan._id === data.item);
-
-                    if (!loadedFromRefetch && (
-                        selectors.general.currentWorkspace(getState()) === WORKSPACE.ASSIGNMENTS ||
-                        currentPreviewId === data.item || currentEditId === data.item)) {
-                        dispatch(planning.api.fetchById(data.item, {force: true}));
-                    }
+            const storedPlan = selectors.planning.storedPlannings(getState())[data.item];
+            const currentEditId = selectors.forms.currentItemId(getState());
+
+            if (get(storedPlan, '_id') !== currentEditId ||
```

Comment:
Same here, use `_.get(data, 'item')` in case the Planning item is not stored in the session's redux store.